src/connection-manager/dialer/dial-request.ts

```diff
@@ -102,10 +102,23 @@ export class DialRequest {
           }
         }
 
+        // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+        if (!conn) {
+          // Notify Promise.any that attempt was not successful
+          // to prevent from returning undefined despite there
+          // were successful dial attempts
+          throw Error('dialAction led to empty object')
```

Comment:
Please wrap the thrown error in `err-code` with an appropriate error code - see examples elsewhere in the codebase for how it's used.

Reply:
changed, took other occurences as an example